README.md

```diff
@@ -42,6 +42,7 @@ issue.
 1. Install a recent D compiler. DCD is tested with 2.066 and LDC 0.14.0.
 1. Run ```git submodule update --init``` after cloning this repository to grab the MessagePack and Datapacked libraries and the parser from DScanner.
```

Comment:
A dub dep. is based on a git tag. A command line would pull the right version of each sub module.
IDK if you see what I mean but there is no correlation between the state of the submodules used by DCD master and the state of the same modules when seen as DUB packages (since they are not git repos).